core/domain/suggestion_registry.py

```diff
@@ -895,6 +895,13 @@ def accept(self, unused_commit_message):
         # Images need to be stored in the storage path corresponding to the
         # question.
         new_image_filenames = self.get_new_image_filenames_added_in_suggestion()
+
+        # Image for interactions with Image Regions in not included as an html
+        # string. This image is as the imagePath in customization args.
+        if question.question_state_data.interaction.id == 'ImageClickInput':
```

Comment:
Just a question. Are there similar issues with e.g. images in multiple-choice, item selection, drag-and-drop question choices? If not, please explain why here.

Reply:
They have ck-editor that will include the reference of images so that those image names can be recieved from the html string.